telemetry/roles/k8s_prometheus/tasks/install_kube_prometheus.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Get K8s namespace
+  ansible.builtin.command: kubectl get ns
+  changed_when: false
+  register: k8s_ns
+
+- name: Get K8s pods
+  ansible.builtin.command: kubectl get pods --all-namespaces
+  changed_when: false
+  register: k8s_pods
+
+- name: Create monitoring namespace
+  ansible.builtin.command:
+    cmd: kubectl create namespace monitoring
+  changed_when: false
+  when: "'monitoring' not in k8s_ns.stdout"
+
+- name: Create directory for temp k8s files
+  ansible.builtin.file:
+    path: "{{ k8s_tmp_dir }}"
+    state: directory
+    recurse: true
+
+- name: Copy kube_prometheus_values.yml file
+  ansible.builtin.template:
+    src: "{{ kube_prometheus_values_file_source }}"
+    dest: "{{ kube_prometheus_values_file_dest }}"
+    mode: "{{ file_mode }}"
+
+- name: Install kube-prometheus stack
```

Comment:
What about the use of `https://docs.ansible.com/ansible/latest/collections/kubernetes/core/helm_module.html`? you can avoid some of the previous tasks if you use this module.

Reply:
As discussed, omnia will support using helm collections instead of commands in next release